enterprise/frontend/src/metabase-enterprise/embedding/selectors.ts

```diff
@@ -0,0 +1,8 @@
+import { getPlan } from "metabase/common/utils/plan";
+import { getSetting } from "metabase/selectors/settings";
+import type { EnterpriseState } from "metabase-enterprise/settings/types";
+
+export const isInteractiveEmbeddingEnabled = (state: EnterpriseState) => {
+  const plan = getPlan(getSetting(state, "token-features"));
+  return plan === "pro-cloud" || plan === "pro-self-hosted";
```

Comment:
Does this cover potentially other plans that have access to this feature? I am thinking (1) Enterprise but also (2) any other legacy plans that should have access to this feature?

I can ask about (2).